src/styles/illustrated-message.scss

```diff
@@ -0,0 +1,166 @@
+@import 'mixins/mixins';
+
+$block: fd-illustrated-message;
+
+.#{$block} {
+  @include fd-reset();
+
```

Comment:
please remove the empty line between 2 includes

Reply:
Then linting won't pass.

```
running pre-commit lint...

> fundamental-styles@0.21.0-rc.20 lint /Users/madrov/coding/sap/fundamental-styles
> npm run lint:style && npm run lint:js


> fundamental-styles@0.21.0-rc.20 lint:style /Users/madrov/coding/sap/fundamental-styles
> stylelint "{src,.storybook}/**/*.(sa|sc|c)ss"


src/styles/illustrated-message.scss
  7:3  ✖  Expected empty line before at-rule   at-rule-empty-line-before
 27:5  ✖  Expected empty line before at-rule   at-rule-empty-line-before
 ```